### PR TITLE
dashing: update

### PR DESCRIPTION
**Pull request by hyuraku**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----
repair the stable resource "ruby_docs_tarball" URL  because of https://github.com/Homebrew/homebrew-core/runs/1404553185?check_suite_focus=true

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]
have done this couple of times, lgtm.

**Comment by chenrui333** [comment]
previous dashing test fix PRs, #60702 #59391

**Comment by chenrui333** [comment]
This is not a right fix.

```
$ /usr/bin/curl -v https://ruby-doc.com
* Rebuilt URL to: https://ruby-doc.com/
*   Trying 108.168.213.2...
* TCP_NODELAY set
* Connected to ruby-doc.com (108.168.213.2) port 443 (#0)
* ALPN, offering h2
* ALPN, offering http/1.1
* Cipher selection: ALL:!EXPORT:!EXPORT40:!EXPORT56:!aNULL:!LOW:!RC4:@STRENGTH
* successfully set certificate verify locations:
*   CAfile: /etc/ssl/cert.pem
  CApath: none
* TLSv1.2 (OUT), TLS handshake, Client hello (1):
* TLSv1.2 (IN), TLS handshake, Server hello (2):
* TLSv1.2 (IN), TLS handshake, Certificate (11):
* TLSv1.2 (OUT), TLS alert, Server hello (2):
* SSL certificate problem: certificate has expired
* stopped the pause stream!
* Closing connection 0
curl: (60) SSL certificate problem: certificate has expired
More details here: https://curl.haxx.se/docs/sslcerts.html
```

**Comment by chenrui333** [comment]
this can be merged after the brew audit PR.

**Comment by mitchblank** [comment]
On 10.14: `curl: (60) SSL certificate problem: certificate has expired`

This is the exact same problem I've had on #67614 -- the 10.14 CI environment seems to have some issue with certain SSL certs appearing expired to it, but the 10.15/11.0 environments are fine!

**Comment by pthariensflame** [comment]
This is now obviated by #67755.

**Comment by hyuraku** [comment]
> This is now obviated by #67755.

Thank you @pthariensflame

**Pull request closed without merging**
